addons/test_mail/tests/test_ui.py

```diff
@@ -5,6 +5,9 @@
 
 @odoo.tests.tagged('post_install', '-at_install')
 class TestUi(odoo.tests.HttpCase):
+    """ This tour is not added to the tests as the Longpolling required to update the message
```

Comment:
Out of context it is not very clear what means "added to the tests". I would clarify "not imported in `__init__` and therefore not ran automatically by the test suite"